include/deal.II/lac/solver_qmrs.h

```diff
@@ -74,86 +94,101 @@ class SolverQMRS : public Solver<VectorType>
   /**
    * Standardized data struct to pipe additional data to the solver.
    *
-   * There are two possibilities to compute the residual: one is an estimate
-   * using the computed value @p tau. The other is exact computation using
-   * another matrix vector multiplication.
+   * The user is able to switch between right and left preconditioning, that means
+   * solving the systems <i>P<sup>-1</sup>A</i> and <i>AP<sup>-1</sup></i> respectively,
+   * using the corresponding parameter. Note, that left preconditioning means to
+   * employ the preconditioned (BiCG-)residual and otherwise the unpreconditioned one.
```

Comment:
I think that "otherwise the unpreconditioned one" is a bit ambiguous. Do you mean

> (BiCG-)residual, and right preconditioning implies the use of the unpreconditioned (BiCG-)residual.

?